source/agora/node/Registry.d

```diff
@@ -576,13 +578,13 @@ unittest
 }
 
 /// Converts a `ZoneConfig` to an `SOA` record
-private SOA fromConfig (in ZoneConfig zone, Domain name, uint serial) @safe
+private SOA fromConfig (in ZoneConfig zone, Domain name) @safe
 {
     return SOA(
         // mname, rname
         Domain.fromString(zone.primary),
         Domain.fromString(zone.soa.email.value.replace('@', '.')),
-        serial,
+        0,
```

Comment:
Add a comment saying it's set from `updateSOA` ? Or better, move `updateSOA` at the end of `fromConfig` ?

Reply:
`fromConfig` is module scope and I don't think it would be good to move it into `ZoneData`. Since `Serial` is not a configuration value, I will add a comment.